spring-cloud-dataflow-configuration-metadata/src/test/java/org/springframework/cloud/dataflow/container/registry/authorization/support/S3SignedRedirectRequestController.java

```diff
@@ -52,6 +57,7 @@ public ResponseEntity<Resource> getManifests(@RequestHeader("Authorization") Str
 	@RequestMapping("/v2/test/s3-redirect-image/blobs/signed_redirect_digest")
 	public ResponseEntity<Map<String, String>> getBlobRedirect(@RequestHeader("Authorization") String token) {
 		if (!"bearer my_token_999".equals(token.trim().toLowerCase())) {
+			logger.info("getBlobRedirect=BAD_REQUEST");
```

Comment:
Please add some human readable context to the log message as the method name is already in the log prefix. Maybe something like `Token did not contain expected bearer signal therefore responding BAD_REQUEST`.